apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx

```diff
@@ -49,30 +50,41 @@ export const JudgeOverview: React.FC = () => {
     variables: { input: { id: id } },
     fetchPolicy: 'no-cache',
   })
-  const resCase = data?.case
 
   const [updateCaseMutation] = useMutation(UpdateCaseMutation)
-  const updateCase = async (id: string, updateCase: UpdateCase) => {
-    const { data } = await updateCaseMutation({
-      variables: { input: { id, ...updateCase } },
-    })
-    const resCase = data?.updateCase
-    if (resCase) {
-      // Do something with the result. In particular, we want th modified timestamp passed between
-      // the client and the backend so that we can handle multiple simultanious updates.
-    }
-    return resCase
-  }
+  const updateCase = useCallback(
+    async (id: string, updateCase: UpdateCase) => {
+      const { data } = await updateCaseMutation({
+        variables: { input: { id, ...updateCase } },
+      })
+      const resCase = data?.updateCase
+      if (resCase) {
+        // Do something with the result. In particular, we want th modified timestamp passed between
+        // the client and the backend so that we can handle multiple simultanious updates.
+      }
+      return resCase
+    },
+    [updateCaseMutation],
+  )
 
   useEffect(() => {
     document.title = 'Yfirlit kröfu - Réttarvörslugátt'
   }, [])
 
   useEffect(() => {
-    if (!workingCase && resCase) {
-      setWorkingCase(resCase)
+    if (!workingCase && data) {
+      setWorkingCase(data.case)
     }
-  }, [workingCase, setWorkingCase, resCase])
+  }, [workingCase, setWorkingCase, data])
+
+  const hasIsolation = (rass: Case): boolean => {
+    const restrictions =
+      rass?.custodyRestrictions !== undefined
+        ? rass.custodyRestrictions
```

Comment:
I don't think we should use custodyRestrictions here. This should always be based on the request that came from the prosecutor regardless of what the judge decided to do later. This should be exactly the same as in the prosecutor overview.

Reply:
I see. I was thinking about the case when the judge had ruled and signed, and would only see the overview. But thats another story i guess.

Reply:
Lol ups the variable name is rass.